source/development/coredump.rst

```diff
@@ -0,0 +1,194 @@
+.. Copyright (C) 2024, Wazuh, Inc.
+
+.. meta::
+  :description: This section contains instructions to configure and collect coredumps for furture analisys
+
+.. _coredumps:
+
+HowTo configure Wazuh agent's OS to generate `coredumps`
+========================================================
+This section contains instructions to customize the diferent OS where Wazuh agent are installed.
+
+Introduction
+------------
+A **core dump** a.k.a **crash-dump** is a snapshot of the memory of a running process. The S.O. where the Wazuh agent resides, it can automatically create a core dump when a serious or unhandled error occurs. Sometimes a crash dump is useful for diagnosing a process that appears to be hanging; The memory dump may reveal information about the cause of the crash.
```

Comment:
Spanish: The OS where the Wazuh...
Remove double subject: [It] can automatically

Reply:
done